app/services/service_catalog/rbac_access.rb

```diff
@@ -0,0 +1,36 @@
+module ServiceCatalog
+  class RBACAccess
+    attr_reader :acl
+    def initialize(resource, verb)
+      @resource = resource
+      @verb     = verb
+      @regexp   = Regexp.new(":(#{@resource}|\\*):(#{@verb}|\\*)")
+      @app_name = ENV["APP_NAME"] || "catalog"
+    end
+
+    def process
+      RBACService.call(RBACApiClient::AccessApi) do |api|
+        @acl = RBACService.paginate(api, :get_principal_access, {}, @app_name).select do |item|
+          @regexp.match(item.permission)
+        end
+      end
+      self
+    end
+
+    def accessible?
+      @acl.any?
```

Comment:
Should `accessible?` and `id_list` be private?  Or force a call to `process` internally if `@acl.nil?` for either of those methods?

Since they're both public it appears they could be called apart from `process` and `@acl` would be nil, or in the case of line 25 return an error on `nil.collect`